src/index.ts

```diff
@@ -1,3 +1,19 @@
-export * from './ppom-controller';
+export type {
+  NativeCrypto,
+  PPOMState,
+  UsePPOM,
+  UpdatePPOM,
+  PPOMInitialisationStatusType,
+  PPOMControllerActions,
+  PPOMControllerInitialisationStateChangeEvent,
+  PPOMControllerEvents,
+  PPOMControllerMessenger,
+} from './ppom-controller';
+export {
+  REFRESH_TIME_INTERVAL,
+  NETWORK_CACHE_DURATION,
+  PPOMInitialisationStatus,
+  PPOMController,
+} from './ppom-controller';
```

Comment:
There should be no breaking changes here. This is the list of existing package-level exports from the `ppom-controller` file.